assets/src/components/detours/diversionPage.tsx

```diff
@@ -458,6 +461,32 @@ export const DiversionPage = ({
               routeDirection={routeDirection || "??"}
             />
           ) : null}
+          {snapshot.matches({
+            "Detour Drawing": { Active: "Changing Duration" },
+          }) ? (
+            <ChangeDuration.Modal
+              onCancel={() =>
+                send({ type: "detour.active.change-duration-modal.cancel" })
+              }
+              onNext={() =>
```

Comment:
nit(non-blocking?): Could we convert `onActivate` to `onDone` and use that rather than `onNext`?

Reply:
Mmm I see! I used the next step because there was already a concept of `nextStepLabel` that I could use for the `fs` label. So I'd have to introduce more props to turn onActivate into a generic button, but I'm open to that

Reply:
Will wait to discuss this further and apply this next week if needed. Making note of it!